Index.html

```diff
@@ -32,8 +32,53 @@ <h2>I’m a software developer</h2>
          <li><a><img src="media/github.svg" alt="github link"></a></li>
          <li><a><img src="media/angellist.svg" alt="angellint link"></a></li>
         </ul>
+
+        <a class="disable-button"><img src="media/Disabled.svg" alt=""></a>
  
       </div>
+      <section class="work-container">
+        <div class="my-work-text">
+          <h2>My Recent Works</h2>
+          <hr>
+       </div>
+        <div class="work-history">
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+          <a><img src="media/Enable.svg" alt="Portfolio"></a>
+        </div>
+      </section>
+      <section class="about-me">
+        <div class="about-me-text">
+          <h2>About me</h2>
+          <p> I can help you build a product , feature or website Look through some of
+             my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+          </p>
+          <button type="submit">Get my resume</button>
+        </div>
+        <div class="skills">
+          <a><img src="media/Works.Inactive.svg" alt="Portfolio"></a>
+          <a><img src="media/frameworks.Inactive.svg" alt="Portfolio"></a>
+          <a><img src="media/Skills.svg" alt="Portfolio"></a>
+        </div>
+        <form class="contact-info">
+          <h2>I'm always interested in hearing about new projects, so if you'd like 
+            to chat please get in touch.</h2>
+          <div class="input-fields">
+            <input type="text" class="full-name" placeholder="Full Name">
+            <input type="email" class="email-address" placeholder="Email Address">
+            <textarea class="details-text" placeholder="Hey I’d like to connect to..."></textarea>
+            <button class="submit">Get in touch</button>
+            <a>nahid.raihan@gmail.com</a>
+          </div>
+        </form>
```

Comment:
- [ ] While you make a great effort to implement the `contact form`, I can not review it because it is not part of this activity.

Reply:
resolved